### ISSUE TITLE
two questions about get_relative_top_filter?

### ISSUE THREAD

**Issue by haiduo** [voidism/DoLa#20]
Thank you for your excellent work.

1.  I think [this operation](https://github.com/voidism/DoLa/blob/11b73b74ec1a72216e3c97c587177d72d8288f8f/dola.py#L113C8-L113C56) is redundant because **“scores”** has already been passed through log_softmax (i.e., "final_logits = final_logits.log_softmax(dim=-1)"). 

2. When you fix min_tokens_to_keep=1, **probs_max is equal to min_thresh**, so you can also omit [**"sorted_logits, sorted_indices = torch.sort(scores_normalized, descending=True)"**](https://github.com/voidism/DoLa/blob/11b73b74ec1a72216e3c97c587177d72d8288f8f/dola.py#L114C9-L115C63).

After debugging, I verified that the results are the same. Perhaps I missed other details.
